app/components/booking_bug_api.rb

```diff
@@ -0,0 +1,12 @@
+class BookingBugAPI
```

Comment:
I think the naming and public interface is a bit confused here. We've got a `BookingBugAPI` object that we've classified as a component. We call it and then internally it instantiates and returns a `ETL::API` object.